packages/frontend/core/src/modules/collection-rules/impls/order-by/updated-at.ts

```
@@ -0,0 +1,33 @@
+import type { OrderByProvider } from '../../provider';
+import type { OrderByParams } from '../../types';
+
+export class UpdatedAtOrderByProvider
+  extends Service
+  implements OrderByProvider
+{
+  constructor(private readonly docsService: DocsService) {
+    super();
+  }
+  orderBy$(
+    _items$: Observable<Set<string>>,
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)